src/sass/_typography.scss

```diff
@@ -31,3 +31,11 @@
   font-size: 14px;
   line-height: 24px;
 }
+
+@mixin heading-1() {
```

Comment:
Not a fan of calling heading-1 when h1 will be matching what it is doing. Just my opinion

Reply:
done